pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -44,7 +44,7 @@ const (
 	preFilter                                 = "PreFilter"
 	preFilterExtensionAddPod                  = "PreFilterExtensionAddPod"
 	preFilterExtensionRemovePod               = "PreFilterExtensionRemovePod"
-	filter                                    = "Filter"
+	Filter                                    = "Filter"
```

Comment:
Do we have a convention that public variables need to put after private variables?

Reply:
not sure. I put it before private ones